kratos/processes/levelset_convection_process.h

```diff
@@ -317,13 +410,18 @@ class LevelSetConvectionProcess
 
     Variable<double>& mrLevelSetVar;
 
+    Variable<array_1d<double, 3 > >& mrConvectVar;
+
     const double mMaxAllowedCFL;
 
     bool mDistancePartIsInitialized;
 
 	const unsigned int mMaxSubsteps;
 
+    const unsigned int mBfeccOrder;
```

Comment:
Shouldn't this be a `bool`.

Reply:
Same as above (in python solver). I will change this to "bool".